policies/entity_collisions.rego

```diff
@@ -0,0 +1,42 @@
+package after_resolution
+
+import rego.v1
+
+# Data structures to make checking things faster.
+entities_names := { obj |
+    group := input.groups[_]
+    group.type = "entity"
+    obj := { "name": group.name, "namespace_prefix": extract_entity_namespace_prefix(group.name)}
+}
+
+# check that matric names do not collide with namespaces
+deny contains entities_registry_collision(description, name) if {
```

Comment:
could we add some tests for entities and metrics?

Reply:
tests added